pkgs/by-name/ad/adrs/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "adrs";
+  version = "0.2.8";
+
+  src = fetchFromGitHub {
+    owner = "joshrotenberg";
+    repo = pname;
```

Comment:
It is not recommended to reference `pname` from behind. ref: https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Thanks, was not aware of that discussion.